dash-pipeline/bmv2/dash_acl.p4

```diff
@@ -17,12 +17,19 @@ match_kind {
 #define LIST_MATCH list
 #define RANGE_LIST_MATCH range_list
 #else
+#ifdef ARCH_BMV2_V1MODEL
 #define LIST_MATCH optional
 #define RANGE_LIST_MATCH optional
+#endif // ARCH_BMV2_V1MODEL
+#ifdef ARCH_DPDK_PNA
+#define LIST_MATCH ternary
```

Comment:
There are no lists of MAC addresses in the current P4 code.  Maybe that is because the P4 code is not yet functionally complete?  I did not write the code that I started from.  Maybe the original author knows?

Reply:
Agreed that currently the only list where LIST_MATCH will use will be a list of IP prefixes and ternery match will do just fine.